Header header

float32[] coeffs
